compiler/src/iree/compiler/Codegen/Common/CPU/CPULowerToUKernels.cpp

```diff
@@ -517,13 +517,13 @@ getFlagForUserAndOperandTypes(IREE::Encoding::EncodingAttr encoding,
   }
 }
 
-static uint32_t getFlagForRole(IREE::Encoding::EncodingRole role) {
-  switch (role) {
-  case IREE::Encoding::EncodingRole::LHS:
+static uint32_t getFlagForIndex(int64_t roleIndex) {
```

Comment:
For consistency, `roleIndex` -> `operandIndex` everywhere.

Reply:
removed all mentioning of 'role' in favor of 'operand'.